libs/services/Converter.py

```diff
@@ -77,13 +90,10 @@ async def convert(self, source, target, data):
         :param data: given attribute value
         :return: obtained value of target attribute
         """
-        try:
-            result = await getattr(self, f'{source}_to_{target}')(data)
-            if result:
-                return result
-            raise DataNotRetrieved(f'Target attribute {target} not available.')
-        except AttributeError:
-            raise ConversionNotSupported(f'Conversion from {source} to {target} is not supported.')
+        result = await getattr(self, f'{source}_to_{target}')(data)
+        if result:
+            return result
+        raise TargetAttributeNotRetrieved(f'Conversion ({self.service_name}) {source} -> {target}: no data retrieved.')
```

Comment:
minor thing: I'd generally put this into an `else` branch to be explicit about how the method works. This syntax can be a bit confusing.

Reply:
Cover in 118cfa5.